pyramid_oereb/lib/renderer/extract/json_.py

```diff
@@ -111,21 +112,25 @@ def _render(self, extract, param):
                 'LogoPLRCadastreRef': self._request.route_url(
                     '{0}/image/logo'.format(route_prefix),
                     logo='oereb',
-                    language=self._language
+                    language=self._language,
+                    extension=filetype.guess_extension(bytearray(extract.logo_plr_cadastre.content)) or 'png'
```

Comment:
Why we use default png here?

Reply:
I could remove the default type and wrap it into a small file type validator. Validation only succeeds for allowed types (png, jpg and svg). Therefore, a matcher for svg files would have to be added.

What do you think?

Reply:
Sounds more proof. How would you design a svg matcher?